etc/setup.sh

```diff
@@ -33,7 +33,8 @@ do_bullseye(){
 	apt-get install -y -t $(grep VERSION_CODENAME /etc/os-release | cut -d= -f2)-backports golang-go
 
 	# Raspberry Pi support
-	grep -q Raspberry /proc/cpuinfo && apt-get install -y wiringpi libpigpio-dev && exit
+	grep -q Raspberry /proc/cpuinfo && apt-get install -y wiringpi libpigpio-dev
+	grep -q Raspberry /proc/cpuinfo && exit
```

Comment:
Drive-by fix. NickS hit a case where if the install fails, it doesn't exit, and then ends up doing the non-pi code below on an actual pi.